CHANGELOG.md

```diff
@@ -1,6 +1,110 @@
 ## 2.15.0 (TBC)
 
-Changes in progress for v2.15.0 are available at [.changes/v2.15.0](https://github.com/vmware/go-vcloud-director/tree/main/.changes/v2.15.0) until the release.
+## FEATURES
+* Added support for Shareable disks, i.e., independent disks that can be attached to multiple VMs which is available from
+  API v35.0 onwards. Also added uuid to the Disk structure which is a new member that is returned from v36.0 onwards. This
```

Comment:
UUID is written in lowercase here an later in capital letters. Let's be consistent with this.

Reply:
Made uppercase for both